README.md

```diff
@@ -0,0 +1,6 @@
+# CocoaPods Podspecs for openwhisk-client-swift
+
+[![License](https://img.shields.io/badge/license-Apache--2.0-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0)
+#[![Build Status](https://travis-ci.org/apache/incubator-openwhisk-podspecs.svg?branch=master)](https://travis-ci.org/apache/incubator-openwhisk-podspecs)
```

Comment:
@mrutkows It appears that travis is not yet set up for this repo. The resulting badge just says "unknown" and clicking it takes me to a Travis page that says the repo is not active.